client/me/purchases/remove-purchase/index.jsx

```diff
@@ -343,7 +345,12 @@ const RemovePurchase = React.createClass( {
 					className="remove-purchase__dialog"
 					isVisible={ this.state.isDialogVisible }
 					onClose={ this.closeDialog }>
-					<FormSectionHeading>{ this.translate( 'Remove %(productName)s', { args: { productName } } ) }</FormSectionHeading>
+					<FormSectionHeading>
+						{ translate( 'Your thoughts are needed.' ) }
```

Comment:
This comment shouldn't have happened either, since the suggestion is the actual string.
Investigating